### DISCUSSION TITLE
make official bounty system on osu

### DISCUSSION THREAD

**Discussion by IdkeMint** [ppy/osu#22319] [Ideas] [answered]
bounties will be on osu site right on main page you just need to make score on specific map or snipe one specific player or something what other player want from anybody for his money, i think what we need to add for this system is verification for players what you shold do for it - make rohulk style liveplay and proof on this liveplay that you are legit(verification works for you only when you are in your rank for example you have verified by support then you farmed from 100k overall to 10k and then you need to verify again)

**Comment by mk56-spn** [comment]
this introduces a considerably large monetary responsibility and headache for the devs at little benefit to anyone since the current bounty system is being run just fine by its manager/s  ( im aware that bountys already exist for non score stuff, but this involves a whole new layer of legitimacy verification ) , also this discussion is likely better suited to the osu web repo

**Comment by peppy** [comment, marked as answer]
Out of scope, will not happen.
